daisy_workflows/export/disk_export_ext.wf.json

```diff
@@ -32,37 +32,50 @@
     }
   },
   "Sources": {
-    "${NAME}_export_disk_ext.sh": "./export_disk_ext.sh"
+    "${NAME}_export_disk_ext.sh": "./export_disk_ext.sh",
+    "disk_resizing_mon.sh": "./disk_resizing_mon.sh"
   },
   "Steps": {
     "setup-disks": {
       "CreateDisks": [
         {
-          "Name": "disk-${NAME}",
+          "Name": "disk-${NAME}-os-${ID}",
           "SourceImage": "${export_instance_disk_image}",
-          "SizeGb": "${export_instance_disk_size}",
-          "Type": "pd-ssd"
+          "Type": "pd-ssd",
+	  "ExactName": true
+        },
+        {
+          "Name": "disk-${NAME}-buffer-${ID}",
+          "SizeGb": "${buffer_disk_size}",
+          "Type": "pd-ssd",
+	  "ExactName": true
         }
       ]
     },
     "run-${NAME}": {
       "CreateInstances": [
         {
           "Name": "inst-${NAME}",
-          "Disks": [{"Source": "disk-${NAME}"}, {"Source": "${source_disk}", "Mode": "READ_ONLY"}],
+          "Disks": [
+            {"Source": "disk-${NAME}-os-${ID}"},
+            {"Source": "${source_disk}", "Mode": "READ_ONLY"},
+            {"Source": "disk-${NAME}-buffer-${ID}"}
+          ],
           "MachineType": "n1-highcpu-4",
           "Metadata": {
             "block-project-ssh-keys": "true",
             "gcs-path": "${OUTSPATH}/${NAME}",
-            "format": "${format}"
+            "format": "${format}",
+	    "instance-name": "inst-${NAME}",
```

Comment:
This appears to be unused, also this is not the actual instance name as it will have a daisy generated name if ExactName isnt set